assets/stylesheets/gu-sass/colours.scss

```diff
@@ -94,7 +94,7 @@ $gu-colours: (
   garnett-neutral-4: #dcdcdc,
   garnett-neutral-5: #ffffff,
   garnett-neutral-6: #767676,
-  garnett-neutral-7: #444444,
+  garnett-neutral-7: #333333,
```

Comment:
This colour was actually updated to `#333` in the dotcom palette on the final hours of the redesign. It's not being used by any components right now so it's safe to change this globally.